### PR TITLE
Add issue Github Action workflow 

### LINKED ISSUE #4: Add a basic Github Action workflow
## Description

CI/CD pipelines for handling automation tasks (e.g. code quality checks & deployment process) are desired.

## Possible Solutions

GitHub Actions is probably the handiest solution. We can implement a basic CI workflow with simple jobs (e.g. lint on PR) first and then add other jobs/workflows on demand.

### PR DESCRIPTION
**Pull request by neo-maker-sudo**
## Related Issue

resolve #7

### CODE REVIEW

**Comment by neo-maker-sudo** [comment]
一直出錯，重複 commit，好醜0.0

File: .github/workflows/package.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (mattwang44) [inline .github/workflows/package.yaml]: Remove the empty line at the top of the file
```suggestion
```

File: .github/workflows/package.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 poetry
```
Comment (mattwang44) [inline .github/workflows/package.yaml]: Always leave a line break at end of the file.

```suggestion
        python -m pip install flake8 poetry

```

File: .github/workflows/package.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+name: Python package
```
Comment (mattwang44) [inline .github/workflows/package.yaml]: This workflow tends to be executed on push or PR; therefore the workflow name `Python package` and the file name `package` does not fit the purpose. Changing it to `ci` or anything related would be more comprehensible.

```suggestion
name: CI
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]
@neo-maker-sudo Great job! Only a few minor issues need to be fixed!

**Comment by mattwang44** [comment]
> 一直出錯，重複 commit，好醜0.0

@neo-maker-sudo 趁這個機會來學學怎麼整理 commit 吧！請你查一下怎麼使用 `git rebase` 這個指令，然後嘗試把**目前的 6 個 commits 合併成 1 個**～

**Comment by mattwang44** [comment]
Another tip is you can [link a PR to an issue with keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). For example, this PR resolves the issue #4 so you can add `resolves #4` in the PR description.

File: .github/workflows/package.yaml (hunk 1 above)
Comment (neo-maker-sudo) [reply .github/workflows/package.yaml]: ok

**Review by mattwang44 — APPROVED** [review]
LGTM!

**Pull request merged**